README.md

```diff
@@ -116,6 +117,26 @@ type DatabaseService = {
 };
 ```
 
+```ts
+type MongoService = {
+  type: 'MONGO';
+  serviceName: string;
+  environment?:
+    | aws.ecs.KeyValuePair[]
+    | ((services: Services) => aws.ecs.KeyValuePair[]);
+  secrets?: aws.ecs.Secret[] | ((services: Services) => aws.ecs.Secret[]);
```

Comment:
```suggestion
  environment?: aws.ecs.KeyValuePair[];
  secrets?: aws.ecs.Secret[];
```
We can maybe simplify envs and secrets for Mongo service because I don't think they will be dependent on other services like web server.